<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibForm</class>
 <widget class="QWidget" name="CalibForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="calibItemsGroupBox">
     <property name="title">
      <string>Calib Items</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="eepromValidLabel">
        <property name="text">
         <string>EEPROM data:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="calibValidLabel">
        <property name="text">
         <string>Calib data:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Unique ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QTableWidget" name="calibItemTableWidget">
        <column>
         <property name="text">
          <string>Calib Item</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>EEP Address</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="readCalibPushButton">
          <property name="text">
           <string>Reread</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="writeEepromPushButton">
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Write EEPROM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="biasCalibGroupBox">
     <property name="title">
      <string>Bias Voltage Calculation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ADC voltage</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Bias Voltage</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="biasAdcLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="biasVoltageLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Formula:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="biasFormulaLabel">
          <property name="text">
           <string>Vbias = VDIV/100 * Vadc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="currentCalibGroupBox">
     <property name="title">
      <string>Bias Current Calculation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
